data/Templates/Hl7v2/Resource/_Encounter.liquid

```diff
@@ -253,6 +286,16 @@
         {
             {% include 'DataType/CWECodeableConcept' mapping: 'CodeSystem/EncounterPriority', CWE: PV2.25 -%}
         },
+        "diagnosis":
+        [
+            {
+                "use":
+                {
+                    {% if DG1.6.Value == "admitting" -%}
+                    {% endif -%}
```

Comment:
Are we expecting a value here?

Reply:
Yes, we expect a value here, the missing value is caused by mapping table records missing.
From my perspective, the condition statement itself can provide information and can be kept as a part of templates.

Reply:
Agree, thanks!